#goal definition
int32 tigerPosition
---
#result definition
int32 observation
---
#feedback
string status
